community/examples/htcondor-pool.yaml

```diff
@@ -65,6 +67,17 @@ deployment_groups:
         email: $(htcondor_configure.central_manager_service_account)
         scopes:
         - cloud-platform
+      network_interfaces:
+      - network: null
+        subnetwork: $(network1.subnetwork_self_link)
+        subnetwork_project: $(vars.project_id)
+        network_ip: ((module.htcondor_configure.central_manager_internal_ips[0]))
```

Comment:
It would help avoid a literal both here and other circumstances where there is only 1 CM but you need to index a 1-element array. What do you think about 2 outputs, 1 with only the first element of the list and the other being the list? e.g.

https://github.com/terraform-google-modules/terraform-google-service-accounts/blob/489cae35180567b3c14b7eda9a05c425acf1b42e/outputs.tf#L17-L41

Other context: there are two daemons on the central manager. In HA mode, one operates hot-hot and the other hot-cold. I don't think there is a way to guide the algorithm for deciding which is hot or cold. So (A) there isn't a primary and a backup (B) even if there were, we wouldn't know at this point in the code.